### ISSUE TITLE
MySQL error when filtering by related elements

### ISSUE THREAD

**Issue by MoritzLost** [masugadesign/cp-filters-craft-cms#15]
**Describe the bug**  

When filtering by any field that contains a relation to other elements (Assets field, Entries field, Categories field etc) I'm getting a MySQL error. Here's the full exception message (the site's table prefix is `bl_`):

```
SQLSTATE[42S22]: Column not found: 1054 Unknown column 'bl_elements.id' in 'field list'
The SQL being executed was: SELECT `bl_elements`.`id`, `bl_content`.`title`, `bl_entries`.`typeId`, `bl_entries`.`sectionId`
FROM (SELECT `elements`.`id` AS `elementsId`, `elements_sites`.`id` AS `elementsSitesId`, `content`.`id` AS `contentId`, `structureelements`.`structureId`
FROM `bl_elements` `elements`
INNER JOIN `bl_entries` `entries` ON `entries`.`id` = `elements`.`id`
INNER JOIN `bl_elements_sites` `elements_sites` ON `elements_sites`.`elementId` = `elements`.`id`
INNER JOIN `bl_content` `content` ON `content`.`elementId` = `elements`.`id`
LEFT JOIN `bl_structureelements` `structureelements` ON (`structureelements`.`elementId` = `elements`.`id`) AND (EXISTS (SELECT *
FROM `bl_structures`
WHERE (`id` = `structureelements`.`structureId`) AND (`dateDeleted` IS NULL)))
WHERE (`entries`.`sectionId`='9') AND (`elements`.`archived`=FALSE) AND (`elements`.`dateDeleted` IS NULL) AND (`elements`.`draftId` IS NULL) AND (`elements`.`revisionId` IS NULL)
ORDER BY `title`) `subquery`
INNER JOIN `bl_entries` `entries` ON `entries`.`id` = `subquery`.`elementsId`
INNER JOIN `bl_elements` `elements` ON `elements`.`id` = `subquery`.`elementsId`
INNER JOIN `bl_elements_sites` `elements_sites` ON `elements_sites`.`id` = `subquery`.`elementsSitesId`
INNER JOIN `bl_content` `content` ON `content`.`id` = `subquery`.`contentId`
LEFT JOIN `bl_structureelements` `structureelements` ON (`structureelements`.`elementId` = `subquery`.`elementsId`) AND (`structureelements`.`structureId` = `subquery`.`structureId`)
ORDER BY `title`
```

Not sure what exectly is causing this issue. I checked the database, the `bl_elements` table is there as expected, as is the `id` column.

**Craft Version & Installed Plugin Version**  

Craft: 3.7.10  
Plugin: 1.1.2

**To Reproduce**  

1. Install Craft 3.7.10 and the CP Filters plugin (1.1.2) 
2. Go to the plugin's entries view and select an entry type to filter by.
3. Filter by any field containing a relation to another element, select 'is assigned' and input any value in the text field. 
4. Click "Refresh results". The error quotes above (or a variation of it) appears.

**Expected behavior**  

Filtering by fields with element relations should not result in an error.

**Additional context**  

Note sure if there's anything wrong with my installation – everything else is working fine. So I'm not sure if the steps to reproduce are complete, but I don't think I have any special plugins or modules that would mess with the database structure or the element queries, so I don't think it's just an issue with my installation.

**Comment by benjaminkohl** [comment]
Would you be able to send me the full stack trace to this error from your Craft web.log file? Feel free to obfuscate system paths or any other sensitive information.

**Comment by MoritzLost** [comment]
@benjaminkohl Sure, I've attached the part of the log containing the exception and the stack trace. Let me know if you need more information or context from the log file!

[web.log](https://github.com/masugadesign/cp-filters-craft-cms/files/7045084/web.log)

**Comment by benjaminkohl** [comment]
We just released CP Filters 1.1.3 with a fix for this. Our queries were applying the DB table name prefix in the column selects but Craft apparently aliases all the table names so applying the DB prefixes was breaking the query.

**Comment by MoritzLost** [comment]
Awesome, thank you @benjaminkohl!
